Fourth_Seminar/app/src/main/java/org/sopt/androidseminar/adapters/RepositoryAdapter.kt

```diff
@@ -36,8 +40,32 @@ class RepositoryAdapter() :
         RecyclerView.ViewHolder(binding.root) {
         fun bind(repositoryInfo: RepositoryInfo) {
             binding.apply {
-                repo = repositoryInfo
+                //repo = repositoryInfo
+                tvRepositoryName.text = repositoryInfo.repoName
+                if (repositoryInfo.repoLang.isNullOrEmpty()) {
+                    tvRepositoryLanguage.text = setSpanString("No Language")
+                } else {
+                    tvRepositoryLanguage.text = repositoryInfo.repoLang
+                }
+                // 만약 레포지토리 설명이 없으면 Github처럼 No Description을 italic체로 보여준다.
+                if (repositoryInfo.repoDescription.isNullOrEmpty()) {
+                    tvRepositoryDetail.text = setSpanString("No Description")
+                } else {
+                    tvRepositoryDetail.text = repositoryInfo.repoDescription
+                }
             }
         }
+
+        private fun setSpanString(str: String): SpannableStringBuilder {
+            val noDescriptionStr = SpannableStringBuilder()
+            noDescriptionStr.append(str)
+            noDescriptionStr.setSpan(
+                StyleSpan(Typeface.ITALIC),
+                0,
+                noDescriptionStr.toString().length,
+                Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
+            )
+            return noDescriptionStr
+        }
```

Comment:
```suggestion
        private fun String.setSpan(): SpannableStringBuilder {
            val noDescriptionStr = SpannableStringBuilder()
            noDescriptionStr.append(this)
            noDescriptionStr.setSpan(
                StyleSpan(Typeface.ITALIC),
                0,
                this.length,
                Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
            )
            return noDescriptionStr
        }
        
        "No Language".setSpan()
```

이런식으로 해도 뭔가 더 깔끔할 것 같기도 하고?

Reply:
그렇네 훨씬 난 듯